qvisor/src/vmspace/host_uring.rs

```diff
@@ -60,61 +58,59 @@ pub fn CopyCompleteEntry() -> usize {
 
 #[inline]
 pub fn HostSubmit() -> Result<usize> {
-    let _count = CopyCompleteEntry();
-        
-    let mut count = 0;
+    if SHARESPACE.config.read().UringIO {
+        let _count = CopyCompleteEntry();
 
-    {
-        let mut uring = URING.lock();
-        let mut sq = uring.submission();
-        let mut submitq = SHARESPACE.uringQueue.submitq.lock();
+        let mut count = 0;
 
-        if sq.dropped() != 0 {
-            error!("uring fail dropped {}", sq.dropped());
-        }
+        {
+            let mut uring = URING.lock();
+            let mut sq = uring.submission();
+            let mut submitq = SHARESPACE.uringQueue.submitq.lock();
 
-        if sq.cq_overflow() {
-            error!("uring fail overflow")
-        }
-        assert!(sq.dropped() == 0, "dropped {}", sq.dropped());
-        assert!(!sq.cq_overflow());
-
-        while !sq.is_full() {
-            let uringEntry = match submitq.pop_front() {
-                None => break,
-                Some(e) => e,
-            };
+            if sq.dropped() != 0 {
```

Comment:
This mixture of if-s and assert-s seems somewhat confusing to me.

Reply:
Remove if-s/